bears/general/KeywordBear.py

```diff
@@ -2,18 +2,71 @@
 
 from coalib.bearlib import deprecate_settings
 from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
 from coalib.results.Result import RESULT_SEVERITY, Result
 
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.results.SourceRange import SourceRange
+
+
+def _get_comments(dependency_results):
+    if not dependency_results:
+        return []
+    annotation_bear_results = dependency_results.get('AnnotationBear')
+    if (not annotation_bear_results or
+            not isinstance(annotation_bear_results, list)):
+        return []
+    comments = []
+    for result in annotation_bear_results:
```

Comment:
ah so this is a workaround for the AnnotationBear not necessarily returning only one result? Can you fix that in the AnnotationBear instead, just adding a return or so would probably be sufficient so we don't need such complicated logic here.